### ISSUE TITLE
Profiling Issue

### ISSUE THREAD

**Issue by PengyuLi7** [sgl-project/sglang-jax#1190]
Why there is only cpu profiling results when I use sglang-jax to implement model inference on TPUs following the instructions provided by sglang-jax? I can only see the CPU timeline for scheduling

**Comment by AmedeoBiolatti** [comment]
I have the same issue on GPU. My current best guess is that some env variable is removed from the scheduler process env

**Comment by PengyuLi7** [comment]
I intend to call the jax.profiler API manually when I launch the inference server, I have no idea whether it works on GPU software stack.



发自我的iPhone


------------------ Original ------------------
From: Amedeo Biolatti ***@***.***&gt;
Date: Sat,May 23,2026 9:18 PM
To: sgl-project/sglang-jax ***@***.***&gt;
Cc: PengyuLi7 ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [sgl-project/sglang-jax] Profiling Issue (Issue #1190)



AmedeoBiolatti left a comment (sgl-project/sglang-jax#1190)
 
I have the same issue on GPU. My current best guess is that some env variable is removed from the scheduler process env
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
